src/zenml/integrations/tekton/orchestrators/tekton_orchestrator.py

```diff
@@ -184,6 +229,70 @@ def prepare_pipeline_deployment(
         )
         deployment.add_extra(ORCHESTRATOR_DOCKER_IMAGE_KEY, repo_digest)
 
+    def _configure_container_op(
```

Comment:
I believe I saw most of this implementation logic somewhere above, would it make sense to abstract some of this into a `BaseKubernetesOrchestrator` perhaps?

Reply:
Well, that's perhaps a story for another time and another PR :D